### PR TITLE
Fix StepBible link document (version)

### LINKED ISSUE #2563: StepBible link uses "version" and not "document"
This does not work:
https://www.stepbible.org/?q=reference=Gen.1.1&document=KJV&v11n=KJV

This works:
https://www.stepbible.org/?q=reference=Gen.1.1|version=KJV&v11n=KJV

Also looks like reference and version must be separated by | and not & (either reference or version can be first)

**Side note**: not sure for what v11n is in this case or if it helps.

**Bug was found on AndBible version**
Build 684 alpha

### PR DESCRIPTION
**Pull request by timbze**
Closes #2563

At least with this it works when our document name matches StepBible's name.
Our NASB is StepBible NASB1995 (they also have NASB2020). There's more differences like that.

TODO
- [x] Update wiki https://github.com/AndBible/and-bible/wiki/Bible-links

### CODE REVIEW

File: app/bibleview-js/src/utils.ts [hunk 1]
```diff
@@ -616,6 +616,6 @@ export function sprintf(format: string, ...args: any[]) {
 }
 
 export function formatExportLink({ref, v11n, doc}: {ref: string, v11n: string, doc?: string}) {
-    const docStr = doc ? `&document=${doc}`: ""
-    return `https://stepbible.org/?q=reference=${ref}&v11n=${v11n}${docStr}`
-}
\ No newline at end of file
+    const docStr = doc ? `|version=${doc}`: ""
+    return `https://stepbible.org/?q=reference=${ref}${docStr}&v11n=${v11n}`
+}
```
Comment (tuomas2) [inline app/bibleview-js/src/utils.ts]: This change is not enough: we need to also intepret this formatting correctly in MainBibleActivity:openLink

Comment (tuomas2) [reply app/bibleview-js/src/utils.ts]: (in case if user clicks link on a device that has andbible set to handle stepbible.org links)

Comment (timbze) [reply app/bibleview-js/src/utils.ts]: I think this is fixed now, but in debug something seems problematic. It seems to start a new AB activity each time and then only opens the reference occasionally. I haven't seen this issue in release build from before though.

**Comment by tuomas2** [comment]
Merging, let's test this!

**Pull request merged**